packages/material-ui/src/FilledInput/FilledInput.js

```diff
@@ -95,12 +98,12 @@ export const styles = theme => {
     },
     /* Styles applied to the `input` element. */
     input: {
-      padding: '27px 12px 10px',
+      padding: '10px 12px',
     },
     /* Styles applied to the `input` element if `margin="dense"`. */
     inputMarginDense: {
-      paddingTop: 24,
-      paddingBottom: 6,
+      paddingTop: [24, '!important'],
+      paddingBottom: [6, '!important'],
```

Comment:
Something's not right here - you shouldn't need to use `important!`.

Reply:
I think its because I put the padding in the root style. I'll look into this on the weekend. I also noticed I didn't account for multiline which uses textarea not input.